corehq/motech/fhir/models.py

```diff
@@ -298,3 +303,29 @@ def get_resource_type_or_none(case, fhir_version) -> Optional[FHIRResourceType]:
         )
     except FHIRResourceType.DoesNotExist:
         return None
+
+
+class FHIRImporter(models.Model):
```

Comment:
food-for-thought: `FHIRImporterConfig`?  Not sure - I was momentarily thrown by what an "Importer" would store in the DB, as I'd expect it to be something responsible for performing the import.

Reply:
:thinking: I went with "FHIRImporter" because it does what `OpenmrsImporter` does ... but ... `OpenmrsImporter` is a Couch `Document`, and when it gets migrated to SQL maybe it should be named "OpenMRSImportConfig".